core/arch/x86/decode_table.c

```diff
@@ -2203,59 +2211,59 @@ const instr_info_t second_byte[] = {
-  {OP_jnz, 0x0f8510, "jnz", xx, xx, Jz, xx, xx, no, fRZ, END_LIST},
-  {OP_jbe, 0x0f8610, "jbe", xx, xx, Jz, xx, xx, no, (fRC|fRZ), END_LIST},
-  {OP_jnbe,0x0f8710, "jnbe",xx, xx, Jz, xx, xx, no, (fRC|fRZ), END_LIST},
+  {OP_jo,   0x0f8010, "jo",  xx, xx, Jz, xx, xx, no, fRO, END_LIST},
+  {OP_jno,  0x0f8110, "jno", xx, xx, Jz, xx, xx, no, fRO, END_LIST},
+  {OP_jb,   0x0f8210, "jb",  xx, xx, Jz, xx, xx, no, fRC, END_LIST},
+  {OP_jnb,  0x0f8310, "jnb", xx, xx, Jz, xx, xx, no, fRC, END_LIST},
+  {OP_jz,   0x0f8410, "jz",  xx, xx, Jz, xx, xx, no, fRZ, END_LIST},
+  {OP_jnz,  0x0f8510, "jnz", xx, xx, Jz, xx, xx, no, fRZ, END_LIST},
+  {OP_jbe,  0x0f8610, "jbe", xx, xx, Jz, xx, xx, no, (fRC|fRZ), END_LIST},
+  {OP_jnbe, 0x0f8710, "jnbe",xx, xx, Jz, xx, xx, no, (fRC|fRZ), END_LIST},
   /* 88 */
-  {OP_js,  0x0f8810, "js",  xx, xx, Jz, xx, xx, no, fRS, END_LIST},
-  {OP_jns, 0x0f8910, "jns", xx, xx, Jz, xx, xx, no, fRS, END_LIST},
-  {OP_jp,  0x0f8a10, "jp",  xx, xx, Jz, xx, xx, no, fRP, END_LIST},
-  {OP_jnp, 0x0f8b10, "jnp", xx, xx, Jz, xx, xx, no, fRP, END_LIST},
-  {OP_jl,  0x0f8c10, "jl",  xx, xx, Jz, xx, xx, no, (fRS|fRO), END_LIST},
-  {OP_jnl, 0x0f8d10, "jnl", xx, xx, Jz, xx, xx, no, (fRS|fRO), END_LIST},
-  {OP_jle, 0x0f8e10, "jle", xx, xx, Jz, xx, xx, no, (fRS|fRO|fRZ), END_LIST},
-  {OP_jnle,0x0f8f10, "jnle",xx, xx, Jz, xx, xx, no, (fRS|fRO|fRZ), END_LIST},
+  {OP_js,   0x0f8810, "js",  xx, xx, Jz, xx, xx, no, fRS, END_LIST},
+  {OP_jns,  0x0f8910, "jns", xx, xx, Jz, xx, xx, no, fRS, END_LIST},
+  {OP_jp,   0x0f8a10, "jp",  xx, xx, Jz, xx, xx, no, fRP, END_LIST},
+  {OP_jnp,  0x0f8b10, "jnp", xx, xx, Jz, xx, xx, no, fRP, END_LIST},
+  {OP_jl,   0x0f8c10, "jl",  xx, xx, Jz, xx, xx, no, (fRS|fRO), END_LIST},
+  {OP_jnl,  0x0f8d10, "jnl", xx, xx, Jz, xx, xx, no, (fRS|fRO), END_LIST},
+  {OP_jle,  0x0f8e10, "jle", xx, xx, Jz, xx, xx, no, (fRS|fRO|fRZ), END_LIST},
+  {OP_jnle, 0x0f8f10, "jnle",xx, xx, Jz, xx, xx, no, (fRS|fRO|fRZ), END_LIST},
   /* 90 */
-  {VEX_EXT, 0x0f9010, "(vex ext 79)", xx, xx, xx, xx, xx, mrm, x, 79},
-  {VEX_EXT, 0x0f9110, "(vex ext 80)", xx, xx, xx, xx, xx, mrm, x, 80},
-  {VEX_EXT, 0x0f9210, "(vex ext 81)", xx, xx, xx, xx, xx, mrm, x, 81},
-  {VEX_EXT, 0x0f9310, "(vex ext 82)", xx, xx, xx, xx, xx, mrm, x, 82},
-  {OP_setz,  0x0f9410, "setz",  Eb, xx, xx, xx, xx, mrm, fRZ, END_LIST},
-  {OP_setnz, 0x0f9510, "setnz", Eb, xx, xx, xx, xx, mrm, fRZ, END_LIST},
-  {OP_setbe, 0x0f9610, "setbe", Eb, xx, xx, xx, xx, mrm, (fRC|fRZ), END_LIST},
-  {OP_setnbe,0x0f9710, "setnbe",Eb, xx, xx, xx, xx, mrm, (fRC|fRZ), END_LIST},
+  {E_VEX_EXT, 0x0f9010, "(e_vex ext 79)", xx, xx, xx, xx, xx, mrm, x, 79},
+  {E_VEX_EXT, 0x0f9110, "(e_vex ext 80)", xx, xx, xx, xx, xx, mrm, x, 80},
+  {E_VEX_EXT, 0x0f9210, "(e_vex ext 81)", xx, xx, xx, xx, xx, mrm, x, 81},
+  {E_VEX_EXT, 0x0f9310, "(e_vex ext 82)", xx, xx, xx, xx, xx, mrm, x, 82},
+  {OP_setz,   0x0f9410, "setz",  Eb, xx, xx, xx, xx, mrm, fRZ, END_LIST},
```

Comment:
More superfluous changes, here and below.  It makes this large diff confusing to read.

Reply:
Reverted. Done.